test/Sentry.Tests/SentryMonitorOptionsTests.cs

```diff
@@ -19,13 +19,13 @@ day of month   1-31
             /	step values
     */
     [Theory]
-    [InlineData("* * * * *")]
-    [InlineData("0 0 1 1 *")]
-    [InlineData("0 0 1 * 0")]
-    [InlineData("59 23 31 12 7")]
-    [InlineData("0 */2 * * *")]
-    [InlineData("0 8-10 * * *")]
-    [InlineData("0 6,8,9 * * *")]
+    [InlineData("* * * * *")]           // Every minute
+    [InlineData("0 0 1 1 *")]           // At midnight on January 1st
+    [InlineData("0 0 1 * 0")]           // At midnight on the first day of the month if it's Sunday
+    [InlineData("59 23 31 12 7")]       // At 23:59 on December 31st if it's Saturday
```

Comment:
typo: `Saturday`/`Sunday`

```suggestion
    [InlineData("59 23 31 12 7")]       // At 23:59 on December 31st if it's Sunday
```

I believe both `0` and `7` are `Sunday` ... and `6` would be `Saturday`